homeassistant/components/smscom/notify.py

```diff
@@ -0,0 +1,121 @@
+"""SMS COM platform for notify component."""
+import logging
+import time
+
+from homeassistant.components.notify import (ATTR_TARGET, PLATFORM_SCHEMA,
+                                             BaseNotificationService)
+from homeassistant.const import CONF_RECIPIENT
+import homeassistant.helpers.config_validation as cv
+from serial import Serial
+import voluptuous as vol
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_PORT = "com_port"
+DEFAULT_PORT = "/dev/ttyUSB0"
+CONF_BAUDRATE = "com_baudrate"
+DEFAULT_BAUDRATE = 115200
+
+PLATFORM_SCHEMA = vol.Schema(
+    vol.All(
+        PLATFORM_SCHEMA.extend(
+            {
+                vol.Optional(CONF_BAUDRATE, default=DEFAULT_BAUDRATE): cv.positive_int,
+                vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.string,
+                vol.Required(CONF_RECIPIENT, default=[]): vol.All(
+                    cv.ensure_list, [cv.string]
+                ),
+            }
+        )
+    )
+)
+
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>